kapitler/01-orientering_og_introduksjon.md

```diff
@@ -17,21 +17,13 @@ arkivdanning – også fagsystemer med saksbehandling.
 
 ## Noark tjenestegrensesnitt
 
-Teknisk grensesnitt mot kjernen er en del av standarden. Noark-4 Web
-Services ble utarbeidet i 2006 etter initiativ fra KS, og som et
-resultat av behov hos kommunene. Formålet var å standardisere og
-forenkle kommunikasjonen mellom de ulike systemene i forvaltningen.
-
-Statens Kartverk og KS gjennomførte i GeoIntegrasjonsprosjektet en
-revisjon av Noark-4 Web Services. Tjenestene i Noark-4 Web services er
-avgrenset ut fra behovene fagsystemene har for å arkivere og gjenfinne
-data.
-
-I nåværende form, og sett i lys av avgrensningene, mangler Noark
-tjenestegrensesnittet funksjonalitet. Et utvidet standardisert
-grensesnitt vil legge til rette for gode, sammenhengende tjenester på
-tvers av virksomhetsgrensene i offentlig sektor. De ulike leverandørene
-behøver ikke utvide tjenestene, eller benytte egne grensesnitt.
+Noark 5-tjenestegrensesnittet (API) er en teknisk standard for kommunikasjon
+mellom saksbehandlingssystemer og Noark-løsninger. Formålet var å standardisere og
+forenkle kommunikasjonen mellom de ulike systemene i forvaltningen. 
```

Comment:
"Formålet var" får leseren til å tro at formålet er endret til noe annet nå.  Burde det være "Formålet er"?  Og formålet er vel å sikre leverandøruavhengig maskinell tilgang til arkivering og gjenfinning av arkivert materiale, ikke standardisering og forenklet kommunikasjon i seg selv?

Foreslår forresten at "er en teknisk standard..." endres til "er en spesifikasjon av en dataprotokoll for maskinell kommunikasjon med Noark-systemer".  Synes "teknisk standard" er for vagt, og bruksområdet "kommunkasjon mellom saksbehandlingssystemer og Noark-løsninger" er smalere enn prosjektets hovedmål.